src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -348,6 +367,15 @@ public bool TargetPlatformSet
         private set;
     }
 
+    /// <summary>
+    /// Gets a value indicating whether default platform is set.
+    /// </summary>
+    public bool DefaultPlatformSet
```

Comment:
Nit: IsDefaultPlatformSet

Reply:
This is following the same naming pattern as the rest of the file.